website/docs/docs/cloud/cloud-cli-installation.md

```diff
@@ -78,6 +78,8 @@ Follow the same process in [Installing dbt Cloud CLI](#manually-install-windows-
 > $ pwd
 /Users/user/dbt-projects/jaffle_shop
 
+> $ echo "project-id: '<your project id>'" > test.yml
```

Comment:
what does `test.yml` mean here?

```suggestion
b. $ echo "project-id: 'your_project_id'" > test.yml ## Input
```

Reply:
This was a mistake on my part! this creates the dbt_cloud.yml file, which I've correctly renamed in my latest commit.